package.json

```diff
@@ -22,45 +19,23 @@
     "eslintconfig",
     "ni"
   ],
-  "author": "NI",
+  "author": "National Instruments",
```

Comment:
Legal name / copyright name should stay National Instruments. Marketing and copy uses NI.